src/middleware/withUcanInvocationHandler.types.ts

```diff
@@ -4,6 +4,8 @@ import { DelegationsStorageContext } from './withDelegationsStorage.types.js'
 import { Service } from '../server/api.types.js'
 import * as Server from '@ucanto/server'
 export interface Environment extends MiddlewareEnvironment {
+  GATEWAY_VALIDATOR_PROOF?: string
+  CONTENT_SERVE_AUTHORITY_PUB_KEY?: string
```

Comment:
Pretty sure this is unused at this point and should be removed -- this was something I think I added at one point